Test create lots of log

when run `github.com/pingcap/pd/server/schedulers`, there will be a lot of log. It will interfer with debug.
```
[2019/11/07 12:29:31.001 +00:00] [ERROR] [region_tree.go:224] ["wrong keys"] [start-key=67] [end-key=67] [stack="github.com/pingcap/log.Error\n\t/go/pkg/mod/github.com/pingcap/log@v0.0.0-20190715063458-479153f07ebd/global.go:42\ngithub.com/pingcap/pd/server/core.(*regionTree).RandomRegion\n\t/go/src/github.com/pingcap/pd/server/core/region_tree.go:224\ngithub.com/pingcap/pd/server/core.(*regionSubTree).RandomRegions\n\t/go/src/github.com/pingcap/pd/server/core/region.go:514\ngithub.com/pingcap/pd/server/core.(*RegionsInfo).RandFollowerRegions\n\t/go/src/github.com/pingcap/pd/server/core/region.go:783\ngithub.com/pingcap/pd/server/core.(*BasicCluster).RandFollowerRegion\n\t/go/src/github.com/pingcap/pd/server/core/basic_cluster.go:160\ngithub.com/pingcap/pd/server/schedulers.(*balanceLeaderScheduler).transferLeaderIn\n\t/go/src/github.com/pingcap/pd/server/schedulers/balance_leader.go:207\ngithub.com/pingcap/pd/server/schedulers.(*balanceLeaderScheduler).Schedule\n\t/go/src/github.com/pingcap/pd/server/schedulers/balance_leader.go:164\ngithub.com/pingcap/pd/server/schedulers.(*testBalanceLeaderRangeSchedulerSuite).TestSingleRangeBalance\n\t/go/src/github.com/pingcap/pd/server/schedulers/balance_test.go:470\nreflect.Value.call\n\t/usr/local/go/src/reflect/value.go:460\nreflect.Value.Call\n\t/usr/local/go/src/reflect/value.go:321\ngithub.com/pingcap/check.(*suiteRunner).forkTest.func1\n\t/go/pkg/mod/github.com/pingcap/check@v0.0.0-20190102082844-67f458068fc8/check.go:836\ngithub.com/pingcap/check.(*suiteRunner).forkCall.func1\n\t/go/pkg/mod/github.com/pingcap/check@v0.0.0-20190102082844-67f458068fc8/check.go:730"]
```
it be created by [line](https://github.com/pingcap/pd/blob/d6d08d9b58fa42a29c3b3f517b9a64a562475588/server/core/region_tree.go#L224)
e.g [ci](https://circleci.com/gh/pingcap/pd/6545?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

--- a/server/core/region_tree.go
+++ b/server/core/region_tree.go
@@ -226,9 +226,11 @@ func (t *regionTree) RandomRegion(ranges []KeyRange) *RegionInfo {
 		}
 
 		if endIndex <= startIndex {
-			log.Error("wrong keys",
-				zap.String("start-key", string(HexRegionKey(startKey))),
-				zap.String("end-key", string(HexRegionKey(endKey))))
+			if len(endKey) > 0 && bytes.Compare(startKey, endKey) > 0 {
+				log.Error("wrong range keys",
+					zap.String("start-key", string(HexRegionKey(startKey))),
+					zap.String("end-key", string(HexRegionKey(endKey))))
+			}
 			continue
 		}
 		index := rand.Intn(endIndex-startIndex) + startIndex

